_pages/location.md

```diff
@@ -0,0 +1,319 @@
+---
+description: DjangoCon Europe 2023 Location.
+heading: Welcome to Edinburgh
+layout: default
+permalink: /location/
+published: true
+title: Location
+---
+
+  <section class="section-pad" id="welcome">
+    <div class="row column">
+      <p class="lead">
+        We can’t wait to welcome you to Edinburgh, <a href="https://www.timeout.com/things-to-do/best-cities-in-the-world">Timeout’s best city in the world 2022</a>!
+      </p>
+    </div>  
+  </section>
+  
+  <section class="section-pad theme-medium-gray">
+  <header class="text-center">
+    <h2>The City</h2>
+  </header>
+    <div class="row column">
+      <div class="medium-6 column">
+      <img
+        class="thumbnail thumbnail-block"
+        src="/static/img/locations/calton-hill.jpg"
+        alt="Calton Hill" />
+      </div>
+      <div class="medium-6 column">
+        <p>
+        Edinburgh is Scotland’s capital city, and the most visited city in the UK after
+        London.  It is perhaps best known as the home of the Edinburgh Fringe and 
+        International Festivals in August each year, but there is plenty to experience at 
+        any time of year.  Many attractions and landmarks are within a few minutes walk of 
+        the conference venue, including Princes Street Gardens, the Scott Monument, the 
+        historic Old Town, the monuments and observatory at Calton Hill, and of course 
+        Edinburgh Castle and the Royal Mile.
+        </p>
+      </div>
+      <div class="column">  
+        <p>Check out the following resources for some guides to the city:
+        <ul>
+          <li><a href="https://www.visitscotland.com/destinations-maps/edinburgh/">Visit Scotland</a></li>
+          <li><a href="https://www.introducingedinburgh.com/">Introducing Edinburgh</a></li>
+          <li><a href="https://edinburgh.org/">Forever Edinburgh</a></li>
+          <li><a href="https://www.timeout.com/edinburgh">Timeout</a></li>
+        </ul>
+        </p>  
+      </div>
+    </div>  
+  </section>
+
+  <section id="weather">
+  <header class="subpage-header">
+    <h2>Weather</h2>
+  </header>
+  <div class="row column">
+    <p>
+      Edinburgh’s weather is reliably unpredictable.  May/June are typically some of the 
+      warmer and drier months of the year, but you should be prepared for any weather!  
+      It’s a good idea to have lightweight waterproof clothes with you, no matter how nice 
+      it looks first thing in the morning!
+      </p>  
+  </div>
+  </section>
+
+
+  <section id="getting-here">
+  <header class="subpage-header">
+    <h2>Getting here</h2>
+  </header>
+  <div class="row column">
+    <h3>By Air</h3>
+    <p>
+      Edinburgh Airport is just 6 miles from the centre, with direct flights to many 
+      destinations around the world. You can easily get to the centre of the city in 
+      around 20 minutes by taxi, bus & tram. You can find more information on getting into 
```

Comment:
Does it make sense to specifically link the airlink buses here? I think a lot of people who are getting here by air might be interested to know that the airport buses are superbly convenient and cost-effective. You've already mentioned the tram which is also a nice option ofc.